meetings/2023-11-15.md

```diff
@@ -0,0 +1,120 @@
+* ...: 1. can be beneficial if access mgmt is done in only one place
+* ...: 2. most apps can avoid interacting with access mgmt
+* ...: by making this simpler, we can make this more secure
+* AC: I agree, it is a high bar to assume that any app will be able to interact with access management system. Also anything what a regular app does can have a profound impacts. Anything what can keep regular apps away from interacting with access policies can be beneficial.
+* eP: What are the best next steps for this? Wait for working group? start discussing now?
+* PAC: I don't think we should wait for the WG. The main reason being that if WG is not chartered to work on that topic it will not pick it up. Given the number of depencencies, including the access control, especially the insufficient maturity level of those specifications. It is worthwhile to have this conversation before the working group starts. We should be clear what to include in the charter.
+* eP: I would propose that we coordinate a special topic meeting to discuss this further
+* ...: should we write something down first and then have the meeting?
+* AC: If you plan a meeting I will arrange to attend it. Outlining some of the specific details would be helpful to focus the conversation.
+
+ACTION: eP will expand on the [related issue](https://github.com/solid/specification/discussions/555) and will propose a date
+
+### W3C Solid WG Charter Proposal
+URL: https://github.com/solid/solid-wg-charter
+
+* PA: We received a [review](https://github.com/w3c/strategy/issues/377) from the TAG, which echo some remarks from the AC review.
+* ...: I don't know why we didn't get this review earlier, before sending it out to AC.
+* ...: Now we are in a strang situation, where what we need to do now is go to the council and _not_ try to resolve existing issues
+* ...: Instead, we should start over with the charter and then go back to the AC with the new charter
+* ...: New idea for the charter: current charter is focused on one part of Solid (client-server protocol), but Solid is much larger than this, including social aspects (client specs, etc). Those social aspects were brought forward in the AC review: trying to solve a very large problem
+* ...: The idea -- reducing the scope to just client to server protocol, leaving aside all the ambitions of building a decentralized web. That's the long term goal
+* ...: What about LDP 2.0: adding access control and authentication
+* ...: making this a successor of LDP would make this less about this being a special way to solve very large problems
+* ...: This also would address the criticism that we are extending something existing rather than specifying something particular from Inrupt
+* eP: This seems like a pragmatic approach to consider. There has been movement to make Solid _not_ a direct successor of LDP
+* ...: A good step might be to document that prior work and how Solid differs from LDP
```

Comment:
@elf-pavlik :

Along the lines of https://solidproject.org/ED/protocol#relationship-to-ldp .. https://github.com/solid/specification/issues/224 ? I am well-aware that there is more to the story.

On 2023-07-03, I wrote the following for that section introducing Solid Protocol's relationship to LDP: https://github.com/solid/specification/pull/539#issue-1785904650

>It is intended to help W3C horizontal reviewers ( e.g., https://github.com/solid/solid-wg-charter/issues/15 ) as well as anyone already familiar with either Solid Protocol or LDP (or both) to understand where they come together or diverge.

See also https://github.com/solid/specification/pull/597/files#r1394432591

/cc @pchampin

Reply:
My understanding (still after re-reading the "Relationship to LDP" section) is that Solid storage servers are complying LDP servers, with some additional restrictions that Solid clients can rely upon. Such restrictions have been introduced by the Solid community for good reasons, so they could be considered either as
* improvements over LDP 1.0, that all LDP 2.0 servers should comply with, or
* a specific profile of LDP which could be specified (but not necessarily required) in LDP 2.0.

Reply:
My last comment on this PR :) We should continue elsewhere.

---

> * improvements over LDP 1.0, that all LDP 2.0 servers should comply with, or

You know.. this is hard. And will take a dedicated Group to do just that. And LDP Next CG would've been the place. Perhaps it could be reopened if there is new interest.

>* a specific profile of LDP which could be specified (but not necessarily required) in LDP 2.0.

I think this would've been good to defer to the [LDP Next CG](https://www.w3.org/groups/cg/ldpnext) for anything that would resemble LDP 2.0, as opposed to Solid Protocol. This is why I proposed to add LDP Next as one of the groups to coordinate with in the WG charter ( https://github.com/solid/solid-wg-charter/pull/12/files#diff-338e19e28f599d3bc3f239af866c9bf1fdcdfdb2e35d7806baffb4bc1ce7ea5bR410-R411 )... and follow-up with the LDP Next CG at the time: https://lists.w3.org/Archives/Public/public-ldpnext/2023Mar/0003.html ... but unfortunately the group recently closed (2023-10-16) : https://www.w3.org/groups/cg/ldpnext/ . This is tiring. I mean.. again, there is even https://fcrepo.github.io/fcrepo-specification/ based off LDP too (and even using/referring to Solid stuff like WAC and LDN.)

I (we?) know that LDP 1.0 doesn't cut it for Solid. We understood that ago, and it only technically / explicitly borrowed some parts. We tried not to introduce any conflicts though. Neither LDP or the Solid Protocol doesn't quite cut it when it comes to utterly simple, and by far the most dominant use case out there e.g., a homepage / a representation of some sort to express an identity (no, I don't mean WebID) / aboutness / general index on the root container (root URL path with /). LDP and Solid Protocol doesn't / clearly explain that out of the box (yet). We needed more implementation experience. In the case of LDP, that wasn't even really taken into account as a UC, AFAIK. In the case of Solid Protocol, some folks seemed to have HTMLphobia and well, we got caught up with conflation of personal preferences with needs and technical feasibility. I've lost track of the number of times where I said: we will not get Solid adopted / or lose a lot of people, if we don't show a clear path on how people can publish/manage something like a homepage using the Solid Protocol consistently, just as with any other resource. And, no, a homepage is not an application for the most widest use - It doesn't not require JavaScript to be human-readable/accessible. That said, I'm confident that this can be resolved.

Any way, the [Solid Protocol Server](https://solidproject.org/ED/protocol#Server) would *generally* be compatible with LDP Server ( https://www.w3.org/TR/ldp/#terms ) - as far as the role is concerned based on HTTP server - but none of the major LDP requirements besides some behaviours related to basic containment. (I don't wish to dive into those details here. As mentioned in the PR that introduced the section, that can all be spelled out further (brief mentions in https://solidproject.org/ED/protocol#server-basic-container and protecting containment statements etc..) Don't get me wrong, there is high overlap and it is relatively easy to extend an implementation conforming to an LDP server to conform to a SP server.

Reply:
@csarven I find your anecdotal case against LDP vague and unconvincing.  Especially as homepage is nowhere mentioned in the charter or your spec, indeed neither is HTML.  I suspect you are lobbying for your minority view position of using RDFa + XHTML.  Conversations in the WebID CG lead me to suspect that will be removed from WebID in favour of JSON-LD.  @elf-pavlik is correct, this should go in a new issue in the charter repo.  There is a path to standardize LDP 2.0 as the basis of Solid 1.0.  Let's examine the options in a proper github issue.

Reply:
Hey @melvincarvalho , nice to see you here. How are things? What you wrote is mega deep, true, and insightful. You should run for chair or table.

Reply:
> Hey @melvincarvalho , nice to see you here. How are things? What you wrote is mega deep, true, and insightful. You should run for chair or table.

@csarven this is completely inappropriate

Reply:
Melvin, please seek my confirmation instead of assuming and sharing your unsolicited speculations and claims regarding my lobbying. Just a hint: you're completely mistaken.

Furthermore, you don't have consent to speak on behalf of or represent the views of the Solid CG (and the WebID CG).

Once more, I encourage you to review the Solid CG's [communication guidelines](https://github.com/solid/specification/blob/main/CONTRIBUTING.md#communication).

Reply:
@csarven this is completely inappropriate.